### PR TITLE
Update/dna gel updates

### PR DESCRIPTION
**Pull request by x-anderson**

### CODE REVIEW

File: an/Liquid_handling/DNA_gel/DNA_gel.an [hunk 1]
```diff
@@ -109,7 +124,7 @@ Steps {
 							//attribute specified mixinpolicy to the DNA ladder
 							Ladder.Type, err = wtype.LiquidTypeFromString(GelLoadingMixingPolicy)
 							if err != nil {
-								Errorf("Error in specifying GelLoadingMixingPolicy %s for DNA Gel: %s", GelLoadingMixingPolicy, err.Error())
+								Ladder.Type, _ = wtype.LiquidTypeFromString(defaultGelLoadingMixPolicy)
```
Comment (chr1sgrant) [inline an/Liquid_handling/DNA_gel/DNA_gel.an]: Some inconsistency between handling an invalid LHPolicy in this protocol, might be better to unify. 
Probably best to add a warning to Errors and set the default but carry on??

Comment (chr1sgrant) [inline an/Liquid_handling/DNA_gel/DNA_gel.an]:
```
err := fmt.Errorf("Error in specifying LoadingDyeMixingPolicy (%s) for DNA Gel. Please assign a valid LHComponent", LoadingDyeMixingPolicy)
Errors = append(Errors, err.Error())

```

File: an/Liquid_handling/DNA_gel/DNA_gel.an [hunk 2]
```diff
@@ -57,21 +58,28 @@ Setup {
...
-	//specify default mixing policy
-	if GelLoadingMixingPolicy == "" {
-		GelLoadingMixingPolicy = "Load"
-	}
-
-	if LoadingDyeMixingPolicy == "" {
-		LoadingDyeMixingPolicy = "NeedToMix"
-	}
+	// //specify default mixing policy
+	// if GelLoadingMixingPolicy == "" {
```
Comment (chr1sgrant) [inline an/Liquid_handling/DNA_gel/DNA_gel.an]: do we need this commented out code?

**Comment by x-anderson** [comment]
depends on [](https://github.com/antha-lang/antha/pull/374)

**Comment by chr1sgrant** [comment]
The parameters and inputs will probably need to be updated in a separate pull request so that they have  correct sentence structure. i.e Start with a capital letter, end in a full stop etc...

**Comment by chr1sgrant** [comment]
@synthacebot test this please

**Pull request merged**